entrypoint.sh

```diff
@@ -20,11 +20,14 @@ SODA_DELFIN_URL=${SODA_DELFIN_URL:-http://127.0.0.1:8190}
 OPENSDS_GELATO_URL=${OPENSDS_GELATO_URL:-http://127.0.0.1:8089}
 OPENSDS_S3_URL=${OPENSDS_S3_URL:-http://127.0.0.1:8090}
 OPENSDS_AUTH_URL=${OPENSDS_AUTH_URL:-http://127.0.0.1/identity}
+SODA_ALERTMANAGER_URL=${SODA_ALERTMANAGER_URL:-http://127.0.0.1:9093}
```

Comment:
hardcoded?
(all these?)

Reply:
These are replaced by the installer at the dashboard installation.
In the installer/ansible/roles/dashboard-installer/scenarios/container.yml the environment variables are exported which are replaced in the nginx config.

But I think I may need to add some more changes to the installer PR as there were some review comments to make the ports configurable.

Let me check that will request review again if all is in order.

Reply:
@skdwriting I have made the following changes to the[ installer PR#413](https://github.com/sodafoundation/installer/pull/413)
![image](https://user-images.githubusercontent.com/19162717/102734641-58c59100-4366-11eb-9da2-07a838568b1b.png)

This will set the `SODA_ALERTMANAGER_URL`  in the env variable at the time of dashboard installation.